UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -624,6 +624,7 @@
 
   gPcAtChipsetPkgTokenSpaceGuid.PcdRtcIndexRegister|$(RTC_INDEX_REGISTER)
   gPcAtChipsetPkgTokenSpaceGuid.PcdRtcTargetRegister|$(RTC_TARGET_REGISTER)
+  gUefiPayloadPkgTokenSpaceGuid.PcdPchBus|0x80

```

Comment:
I'm concerned that this makes UefiPayload Intel-specific. Besides, is this correct?

Reply:
Agree, we should remove Intel-specific definition from DEC and use library for Intel-specific implementation so different soc/platform may have their own library instance.

Reply:
Later, I read that this really refers to a device implementing ISA, which I believe is architectural to x86 (or, close enough). And based on the code, it may be within the context of the FDT/UPL spec, which I haven't read much of, rather than a PCI bus number

I think renaming it to start is good, then considering if it's a spec-defined constant or information that can be read from the bootloader or obtained at runtime. If not, then maybe a library class approach is the way to go

Reply:
we may consider to use HOB for supporting different parent bus/dev/func of serial device.

Reply:
Done.

Reply:
both @benjamindoron and I tried to resolve those comments that are done but there is no option to do that. Maybe maintainers like @gguo11837463 could help?